panel/links.py

```diff
@@ -655,3 +655,11 @@ def _get_code(self, link, source, src_spec, target, tgt_spec):
 Link.register_callback(callback=JSLinkCallbackGenerator)
 
 Viewable._preprocessing_hooks.append(Callback._process_callbacks)
+
+__all__ = [
```

Comment:
I'm in doubt here what you want to expose as part of public api to users of the package as I've very rarely used this functionality/ module.

Reply:
Yeah this really is mostly used internally and should probably be moved to a core module.